backend/app/Actions/ErrorReports/AddErrorReportsAction.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Actions\ErrorReports;
+
+use App\Entities\Error;
+use App\Entities\Page;
+use App\Repositories\Contracts\ErrorReports\ErrorReportsRepository;
+use App\Repositories\Contracts\PageRepository;
+use App\Repositories\Contracts\WebsiteRepository;
+use Tymon\JWTAuth\Facades\JWTAuth;
+use Illuminate\Support\Str;
+
+final class AddErrorReportsAction
+{
+    private $errorReportsRepository;
+    private $websiteRepository;
+    private $pageRepository;
+
+    public function __construct(
+        ErrorReportsRepository $errorReportsRepository,
+        PageRepository $pageRepository,
+        WebsiteRepository $websiteRepository
+    ) {
+        $this->errorReportsRepository = $errorReportsRepository;
+        $this->websiteRepository = $websiteRepository;
+        $this->pageRepository = $pageRepository;
+    }
+
+    public function execute(AddErrorReportsActionRequest $request)
+    {
+        $visitorId = null;
+        if ($request->token()) {
+            JWTAuth::setToken(Str::after($request->token(), 'Bearer '));
+            $visitorId = JWTAuth::getPayload()->get('visitor_id');
+        }
+
+        $websiteId = $this->websiteRepository
+            ->getByTrackNumber($request->trackingNumber())->id;
```

Comment:
If you have visitor, it means you can retrive website by visitor, you do not need a tracking number